transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClient.kt

```diff
@@ -191,4 +191,12 @@ interface MessagingClient {
      * latest available history.
      */
     fun invalidateConversationCache()
+
+    /**
+     * Notify the Agent that the User is typing a message.
```

Comment:
They use term `user` as well, but indeed - `customer` is used more often. I will update the kdocs and keep it in mind for future documentation.

Reply:
Also, while changing from `user` to `customer` I realized that our backend use term `user` for typing indicator fields in deploymentConfig. So now I am a little ambiguous regarding what term should be used.
@crumpf wonder if you think that is fine to use term `customer` for KDoc and other documentation, but internally use the term `User` for cases like UserTypingProvider and similar?

Reply:
It's so tricky with these terms. `Customer` from what I've been imagining in there scenarios would be like Quicken. `End user` or just `User` would be the person using the application. Of course, if there's prior art in Web Messaging already that's tricky but it feels like we should definitely standardize.

Reply:
Ugh. @bjdupuis is right the nomenclature can be difficult. Now I'm having flashbacks to some of the user research studies we were a part of 😬. I agree that we should standardize on these terms, also with the DX mobile, Web Messenger, Messenger, et al. teams.

@AfanasievAnton I'm fine using `user` internally, especially if it helps match terms in the backend use to avoid confusion. Maybe for now, given the prior art in the Web Messaging documentation, we use "user" internally and "customer" in the docs, like you proposed, and try to keep "user" or "customer" out of the SDKs public API unless the user is ambiguous.